### ISSUE TITLE
Total Collection Value [sf#81]

### ISSUE THREAD

**Issue by codeguru42** [BaseballCardTracker/bbct#100]
Automatically total the value of the card collection and display the total to the user.

**Comment by AtharvKinage** [comment]
@codeguru42  I would like to work on this. Is this issue still open? If yes can you please assign this to me.

**Comment by codeguru42** [comment]
@AtharvKinage Yes, you are welcome to work on this. Be sure to fork this repo and create a new branch. When you have something to share with me, create a pull request back to this repo and we can discuss your changes from there and hopefully work towards merging it in.
